### PR TITLE
docs: update readme with more setup details

### PR DESCRIPTION
**Pull request by MichaelGoberling**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Review by shazron — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ governing permissions and limitations under the License.
 # Template Registry API
 
 ## OpenAPI Schema
-Template Registry API follows OpenAPI 3.0.3 Specification.
+Template Registry API follows OpenAPI 2.0 Specification.
```
Comment (arjuncooliitr) [inline README.md:19]: Are we switching back to  OpenAPI 2.0 version?

Comment (MichaelGoberling) [reply README.md:19]: unfortunately yes, we have to because runtime doesn't support openapi 3.0 for the scope and client validation features yet ☹️

Comment (MichaelGoberling) [reply README.md:19]: this pr https://github.com/adobe/adp-template-registry-api/pull/37 reverts us back to 2.0

Comment (arjuncooliitr) [reply README.md:19]: Ahh damn, gotcha

**Review by arjuncooliitr — APPROVED** [review]

**Pull request merged**